doc/cephadm/certmgr.rst

```diff
@@ -16,27 +16,24 @@ while offering flexibility for deployment preferences.
 Certificate Management Behavior
 ===============================
 
-Cephadm `certmgr` automatically detects whether a certificate is self-signed
-(generated by Cephadm) or user-provided. This distinction determines how it
-handles expirations and renewals:
+Cephadm ``certmgr`` automatically detects whether a certificate is self-signed (generated
+by Cephadm) or user-provided as an embedded value in the spec, or referenced externally.
+This distinction determines how it handles expirations and renewals:
 
 Self-Signed Certificates
 ------------------------
-- `certmgr` can fully automate renewal, ensuring seamless service operation.
+- ``certmgr`` can fully automate renewal, ensuring seamless service operation.
 - Automation is controlled by configuration parameters defining certificate
   duration, renewal thresholds, and whether automated rotation is enabled.
 
 User-Provided Certificates
 --------------------------
-- `certmgr` does not renew these automatically but continuously monitors
-  their status.
-- If an issue is detected (e.g., expiration is approaching), it issues
-  health warnings/errors `CEPHADM_CERT_ERROR` to alert administrators,
-  who must manually replace the certificate.
 
-This proactive approach ensures secure and uninterrupted service operations
-while allowing users to manage certificate policies according to their needs.
+There are two types:
+- Inline: Certificates embedded in the service spec; cannot be modified directly via certmgr.
```

Comment:
An empty line is needed here before the unordered list (between lines 32-33) otherwise the list is rendered as a single paragraph.
```suggestion
There are two types:

- Inline: Certificates embedded in the service spec; cannot be modified directly via certmgr.
```

Also in this list the 2 occurrences of "certmgr" is rendered as normal text while in previous text it was rendered as inline code using double backticks, is that intentional?